### PR TITLE
This PR does a number of things:

### PR DESCRIPTION
**Pull request by brian-avery**
* update the version of Hugo that we're using to build our site
* clean up the maistra.io README and document the Makefile flags
* add a verification option to the makefile
* add a verified platforms column to the release table
* document releases in a yaml file that can be used across tooling
* Fix OpenShift links
* Fix issues discovered in Hugo 0.55 https://discourse.gohugo.io/t/misunderstanding-of-shortcode-syntax-in-0-55/18538

### CODE REVIEW

File: layouts/partials/maistraBranch.html [hunk 1]
```diff
@@ -0,0 +1 @@
+{{ (index .Site.Data.releases .Site.Data.release.maistraVersion ).maistraBranch }}
```
Comment (jwendell) [inline layouts/partials/maistraBranch.html:1]: Is this correct?

**Review by dmitri-d — APPROVED** [review]

File: layouts/partials/maistraBranch.html (hunk 1 above)
Comment (brian-avery) [reply layouts/partials/maistraBranch.html:1]: what do you think is wrong with it? It's working on 1.2

Comment (jwendell) [reply layouts/partials/maistraBranch.html:1]: maistraVersion vs maistraBranch

Comment (brian-avery) [reply layouts/partials/maistraBranch.html:1]: to clarify, what this is doing is: 

* `.Site.Data.releases` looks in the `/data/releases` directory. 
* `.Site.Data.release.maistraVersion` looks up the default maistraVersion in the `/data/release` file describing the current release. 
* `index .Site.Data.releases .Site.Data.release.maistraVersion` then grabs the object at that index, which is the file describing the current release. 
* Finally, it looks at the `maistraBranch` entry in that file.

Comment (jwendell) [reply layouts/partials/maistraBranch.html:1]: thanks, sorry for the noise, I thought it was a typo... :)

**Review by jwendell — APPROVED** [review]

File: layouts/partials/maistraBranch.html (hunk 1 above)
Comment (brian-avery) [reply layouts/partials/maistraBranch.html:1]: No problem. Hugo syntax is a bit different.

**Pull request merged**